src/LocalMixedGraphSpectralTargets.hpp

```diff
@@ -90,6 +92,7 @@ class UpscaleParameters
     UpscaleParameters() : max_levels(2),
         spect_tol(0.001),
         max_evects(4),
+        max_traces(4),
```

Comment:
Is this the right default? If you make it 1 then the tests in `stest.py` don't have to be changed as much.

Reply:
I also thought setting it to be 1 would require less change, but actually in that case 27 tests failed while using 4 as default only failed 14 tests.